cli/azd/internal/appdetect/docker.go

```diff
@@ -1,19 +1,75 @@
 package appdetect
 
 import (
+	"bufio"
+	"fmt"
 	"io/fs"
+	"log"
+	"os"
 	"path/filepath"
+	"strconv"
 	"strings"
 )
 
 func detectDocker(path string, entries []fs.DirEntry) (*Docker, error) {
 	for _, entry := range entries {
 		if strings.ToLower(entry.Name()) == "dockerfile" {
+			dockerFilePath := filepath.Join(path, entry.Name())
+			ports, err := parsePortsInFile(dockerFilePath)
+			if err != nil {
+				return nil, fmt.Errorf("parsing ports in file %s: %w", dockerFilePath, err)
```

Comment:
We should just return without error formatting here:

```suggestion
				return nil, err
```

In the current implementation, this looks like:

```bash
parsing ports in file testdata/Dockerfile/DockerfileSinglePort: reading Dockerfile at testdata/Dockerfile/DockerfileSinglePort
```

which isn't quite right.

Reply:
Updated.